### PR TITLE
change modality in upload_manifest if fib is being run or not

### LINKED ISSUE #793: sessions have mislabeled metadata on modalities
The metadata on modalities (which is used to query for sessions on mongoDB) isn't always correct. 

Per @alexpiet , the modality `fib` is generated automatically if there is a `fiber_photometry_start_time` variable in the main GUI window. This makes sense since the sessions that have the wrong metadata are trained in 447 but finish in the EPHYS rig afterwards. 

The bug may also be in `src/foraging_gui/GenerateMetadata.py`

The fix should be done when the data is generated. we need to make sure that a fiber folder is created to have the `fib` modality. 

Examples of this include: 717263, 714116. 

Fib modality listed here: 
Image

But data doesn't have fiber photometry data

Image

On streamlit we see that it's finished trained on the 323 ephys rig 
Image

### PR DESCRIPTION
**Pull request by micahwoodard**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- Update the upload manifests to only list "fib" and "behavior-videos" if they are actually recorded in the session
### What issues or discussions does this update address?
- resolves #793 
### Describe the expected change in behavior from the perspective of the experimenter
- none
### Describe any manual update steps for task computers
- none
### Was this update tested in 446/447?
- [x] 446/7

### CODE REVIEW

**Comment by micahwoodard** [comment]
Still unclear how behavior videos are being set to record in gui but for now, they are included if fib is being run

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -4616,6 +4616,7 @@ def _generate_upload_manifest(self,FIP=False):
             if not hasattr(self, 'project_name'):
                 self.project_name = 'Behavior Platform'
 
+            # FIXME: How is this if statement working if it does the exact same thing for each condition?
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:4619]: I think we had originally planned on triggering different code ocean capsules based on data modalities (FIP), but now we just trigger the same capsule. So we could remove the entire `FIP` variable in this function

File: src/foraging_gui/Foraging.py [hunk 2]
```diff
@@ -4625,6 +4626,18 @@ def _generate_upload_manifest(self,FIP=False):
                 capsule_id = 'c089614a-347e-4696-b17e-86980bb782c1'
                 mount = 'FIP'
 
+            # only include fib and behavior videos if fib is being run
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:4629]: We need to figure out if there is video separately from the FIB

**Review by alexpiet — APPROVED** [review]

**Comment by alexpiet** [comment]
@micahwoodard I renamed the PR for future clarity

**Pull request merged**